src/NUnitEngine/Addins/addin-tests/VisualStudioProjectLoaderTests.cs

```diff
@@ -62,22 +62,22 @@ public void CannotLoadWebProject()
-        [TestCase("vb-sample.vbproj", new string[] { "Debug", "Release" }, "vb-sample")]
-        [TestCase("jsharp-sample.vjsproj", new string[] { "Debug", "Release" }, "jsharp-sample")]
-        [TestCase("fsharp-sample.fsproj", new string[] { "Debug", "Release" }, "fsharp-sample")]
-        [TestCase("cpp-sample.vcproj", new string[] { "Debug|Win32", "Release|Win32" }, "cpp-sample")]
-        [TestCase("cpp-default-library.vcproj", new string[] { "Debug|Win32", "Release|Win32" }, "cpp-default-library")]
-        [TestCase("legacy-csharp-sample.csproj", new string[] { "Debug", "Release" }, "csharp-sample")]
-        [TestCase("legacy-csharp-hebrew-file-problem.csproj", new string[] { "Debug", "Release" }, "HebrewFileProblem")]
-        [TestCase("legacy-vb-sample.vbproj", new string[] { "Debug", "Release" }, "vb-sample")]
-        [TestCase("legacy-jsharp-sample.vjsproj", new string[] { "Debug", "Release" }, "jsharp-sample")]
-        [TestCase("legacy-cpp-sample.vcproj", new string[] { "Debug|Win32", "Release|Win32" }, "cpp-sample")]
-        [TestCase("legacy-cpp-library-with-macros.vcproj", new string[] { "Debug|Win32", "Release|Win32" }, "legacy-cpp-library-with-macros")]
-        [TestCase("legacy-cpp-makefile-project.vcproj", new string[] { "Debug|Win32", "Release|Win32" }, "MakeFileProject")]
+        [TestCase("csharp-sample.csproj", new[] { "Debug", "Release" }, "csharp-sample")]
+        [TestCase("csharp-sample.csproj", new[] { "Debug", "Release" }, "csharp-sample")]
+        [TestCase("csharp-missing-output-path.csproj", new[] { "Debug", "Release" }, "MissingOutputPath")]
+        [TestCase("csharp-xna-project.csproj", new[] { "Debug|x86", "Release|x86" }, "XNAWindowsProject")]
+        [TestCase("vb-sample.vbproj", new[] { "Debug", "Release" }, "vb-sample")]
+        [TestCase("jsharp-sample.vjsproj", new[] { "Debug", "Release" }, "jsharp-sample")]
+        [TestCase("fsharp-sample.fsproj", new[] { "Debug", "Release" }, "fsharp-sample")]
+        [TestCase("cpp-sample.vcproj", new[] { "Debug|Win32", "Release|Win32" }, "cpp-sample")]
+        [TestCase("cpp-default-library.vcproj", new[] { "Debug|Win32", "Release|Win32" }, "cpp-default-library")]
+        [TestCase("legacy-csharp-sample.csproj", new[] { "Debug", "Release" }, "csharp-sample")]
+        [TestCase("legacy-csharp-hebrew-file-problem.csproj", new[] { "Debug", "Release" }, "HebrewFileProblem")]
+        [TestCase("legacy-vb-sample.vbproj", new[] { "Debug", "Release" }, "vb-sample")]
+        [TestCase("legacy-jsharp-sample.vjsproj", new[] { "Debug", "Release" }, "jsharp-sample")]
+        [TestCase("legacy-cpp-sample.vcproj", new[] { "Debug|Win32", "Release|Win32" }, "cpp-sample")]
+        [TestCase("legacy-cpp-library-with-macros.vcproj", new[] { "Debug|Win32", "Release|Win32" }, "legacy-cpp-library-with-macros")]
+        [TestCase("legacy-cpp-makefile-project.vcproj", new[] { "Debug|Win32", "Release|Win32" }, "MakeFileProject")]
```

Comment:
As before, not sure which compilers support and don't support this. For that reason we tend to be conservative. Should be discussed. Maybe we need to add something about such features to the coding standard - or a separate document - so that people don't have to learn it by trial and error.